packages/integration-tests/tests/sign-in-experience.test.ts

```diff
@@ -36,4 +46,45 @@ describe('admin console sign-in experience', () => {
 
     expect(updatedSignInExperience).toMatchObject(newSignInExperience);
   });
+
+  it('should be able to setup sign in methods after connectors are enabled', async () => {
+    // Setup connectors for tests
+    await Promise.all([
+      updateConnectorConfig(facebookConnectorId, facebookConnectorConfig),
+      updateConnectorConfig(twilioSmsConnectorId, twilioSmsConnectorConfig),
+      updateConnectorConfig(sendgridEmailConnectorId, sendgridEmailConnectorConfig),
+    ]);
+
+    await Promise.all([
+      enableConnector(facebookConnectorId),
+      enableConnector(twilioSmsConnectorId),
+      enableConnector(sendgridEmailConnectorId),
+    ]);
+
+    // Set up sign-in methods
+    const newSignInMethods: Partial<SignInMethods> = {
+      username: SignInMethodState.Primary,
+      sms: SignInMethodState.Secondary,
+      email: SignInMethodState.Secondary,
+      social: SignInMethodState.Secondary,
+    };
+
+    const updatedSignInExperience = await authedAdminApi
+      .patch('sign-in-exp', {
+        json: {
+          socialSignInConnectorTargets: [facebookConnectorTarget],
+          signInMethods: newSignInMethods,
+        },
+      })
+      .json<SignInExperience>();
+
+    expect(updatedSignInExperience.signInMethods).toMatchObject(newSignInMethods);
+
+    // Reset connectors
+    await Promise.all([
+      disableConnector(facebookConnectorId),
+      disableConnector(twilioSmsConnectorId),
+      disableConnector(sendgridEmailConnectorId),
+    ]);
```

Comment:
better do it in `afterAll`？

Reply:
Connectors are used in this test only; it doesn't make sense to do the reset logic in `afterAll`.